chain/client/src/sync/state.rs

```diff
@@ -1466,6 +1504,46 @@ pub fn external_storage_location(
     )
 }
 
+/// multi node dump: Construct a location on the external storage.
+pub fn external_storage_location_multi_node(
+    chain_id: &str,
+    epoch_id: &EpochId,
+    epoch_height: u64,
+    shard_id: u64,
+    part_id: u64,
+    num_parts: u64,
+) -> String {
+    format!(
+        "multi_node_tmp_1/{}/{}",
+        location_prefix_with_epoch_id(chain_id, epoch_height, epoch_id, shard_id),
```

Comment:
Why do you need a separate function for `with_epoch_id`? Shouldn't this be made the default? As discussed not including epoch_id in the path is a bad idea because of possible forks.